### ISSUE TITLE
Proxy Support

### ISSUE THREAD

**Issue by fizxmike** [zedapp/zedrem#15]
I am launching my zedrem server on a computer at work which is behind a firewall+proxy. The proxy at my work allows https (port 443) pass-through. I am trying to connect using zed on my chromebook from home.

I am thinking about modifying https://github.com/zedapp/zedrem/blob/cf2d96f254e00f80e3d323c2d88bb39f13aab26b/client.go#L461

To include the automatic detection and utilization of a proxy server using pieces of the technique I found here (I would be querying environment variables http_proxy and https_proxy):
https://gist.github.com/madmo/8548738

I'm assuming that as long as I return a valid websocket.Conn object, things will work as expected. Is this a reasonable assumption?

Are there other connections set up elsewhere in the code that also need to be modified to use a proxy?

General thoughts?

**Comment by zefhemel** [comment]
Yes, if a valid `websocket.Conn` is returned all should work as expected. This would be a very nice addition, by the way. I hope you can get it to work! The zedrem client (presumably, this is what you meant when you mentioned running it at work) only sets up a websocket connection, that's it.
